06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndex.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   * @param {Array} filenames names of files to initialize the index with
+   * @param {Object} files file index(es) to be stored initially
+   */
+  constructor(filenames = [], files = {}) {
+    this.filenames = filenames;
+    this.files = files;
+  }
+
+  /**
+   * Validates file content
+   * @function
+   * @param {Array} fileContent content of uploaded file
+   * @return {Boolean} true if a valid file was uploaded, false otherwise
+   */
+  static validateFile(fileContent) {
+    if ((fileContent instanceof Array) && fileContent.length > 0) {
+      const BreakException = {};
```

Comment:
Thanks, essentially if the error is not want we defined then the default error should be thrown. I have add this.